enhancements/baremetal-instance-types/README.md

```
@@ -0,0 +1,749 @@
+   ```
+2. The fulfillment service updates `spec.state: Deprecated` and records deprecation metadata on
+   the CRD.
+3. The fulfillment service begins returning a warning on `CreateBareMetalInstance` requests
+   referencing this type: "Instance type 'hpc-icelake-96c' is deprecated and will become obsolete
+   on 2027-03-31. Consider migrating to 'hpc-sapphirerapids-128c'."
+4. Existing `BareMetalInstance` and `BareMetalPool` resources continue to operate unchanged.
+
+**Obsoleting an Instance Type**
+
+1. Cloud Provider Admin sets the type to Obsolete:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)